plugins/field-date/test/field_date_test.mocha.js

```diff
@@ -97,7 +119,7 @@ suite('FieldDate', function() {
       });
       test('New Value', function() {
```

Comment:
I know all of the validater tests already existed, but could you update them to have some more informative names? Not a high priority though if you just want to get this merged!

Reply:
Done!